Impossible-Agent-Suite/staxmigrate/README.md

```diff
@@ -0,0 +1,28 @@
+# Impossible Finance staxMigrate mint agent
+
+## Description
+
+This agent detects when Impossible Finance tokens are minted through the function `staxMigrate` where the amount of `STAX` tokens input are not the same as the amount of `IF` tokens output
+
+## Supported Chains
+
+- Ethereum
+- BSC
+
```

Comment:
based on the documentation, Ploygon is a supported chain as well

Reply:
I'm unsure if I should event state that we support the Ethereum chain now. After reviewing Lazaro's PRs he states that BSC is the only supported chain in his READMEs. Thoughts @e1Ru1o ?

Reply:
Yes use only BSC

Reply:
Addressed in latest commit.